### ISSUE TITLE
Guide for Plugin.

### ISSUE THREAD

**Issue by naveenT1010** [xuhao1/Unreal-ROS-Plugin#3]
Hi,
Is there is any guide for implementing this with ROS and Unreal? It seems promising but I am not able to figure out how to set this up.

**Comment by krishenry** [comment]
Hi,
Were you able to solve this problem and implement this system?

**Comment by naveenT1010** [comment]
Yes, I was finally able to use ROS with Unreal.

**Comment by krishenry** [comment]
Hi again and thank you for answering,
Would you be able to give some instructions on how to do this? I am just a beginner using linux and terminal and i want to interface ros and ue4 to control a baxter robot. 
I would really appreciate the help.
Thank you already,
Kris

**Comment by naveenT1010** [comment]
Actually, I didn't do it with this package. There is no clear documentation with this one, and also it seems quite obsolete. Instead I used [UnrealPythonPlugin](https://github.com/20tab/UnrealEnginePython).
It allows you to embed python to unreal, but by default it points to python3.5 so I had to change the default python path of the plugin (instruction of which is given in their readme). So finally I was able to run python scripts in unreal and was able to import rospy.
